frontend/components/Header.tsx

```diff
@@ -38,3 +34,27 @@ const mapStateToProps = (state: AppStore): HeaderProps => {
 	};
 };
 export default connect(mapStateToProps)(Header);
+
+const StyledNav = styled.nav`
```

Comment:
Can you put the styles before the components in every file? Since the components require the styles it makes a little bit more sense to have the variable be declared before its use.

It also doesn't allow you to conveniently stuff the style at the end of a file, and serves as a good rule of thumb when to refactor things into smaller pieces, when the style grows too long :)